client/src/scss/partials/_decoration-workspace.scss

```diff
@@ -0,0 +1,60 @@
+.workspace__steam--1 {
+    @include moveAndFadeAnimation(10px, 6s, .2s);
```

Comment:
I don't think this will work as the animation you intend. Since you are including it as a mixin, the `@keyframes move-and-fade` gets added to the CSS output each time. Although that means you are setting a different `$distance` each time you include it, the animation name does not change since it is hardcoded at https://github.com/istrategylabs/paw/pull/36/files#diff-ba322bcde2e5de2e39a7c20a3c22110dR6. Therefore, the CSS cascade kicks in and the last inclusion will be the animation's final form.